filewatcher/filewatcher.go

```diff
@@ -20,6 +20,9 @@ import (
 // caller can tweak its value when needed.
 var InitialReadInterval = time.Second / 2
 
+// DefaultMaxFileSize is the default MaxFileSize used when it's <= 0.
+const DefaultMaxFileSize = 1024 * 1024 // 1MiB
```

Comment:
Maybe a comment to reference why this is set to 1MiB:
https://lucene.apache.org/solr/guide/7_4/setting-up-an-external-zookeeper-ensemble.html
> ZooKeeper is designed to hold small files, on the order of kilobytes. By default, ZooKeeper’s file size limit is 1MB. Attempting to write or read files larger than this will cause errors.

Reply:
So filewatcher is general purpose and not limited to only watch files synced down from zookeeper, that's why it's a configuration, not just a hardcoded value :)

Reply:
That's why I thought having the comment here and not in the `Config` struct. I didn't know that Zookeeper has a file limit.